openupgrade_scripts/scripts/account/14.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,648 @@
+account      / account.bank.statement.line / journal_id (many2one)         : now required
+# NOTHING TO DO: These fields now are in account.move
+
+account      / account.bank.statement.line / note (text)                   : DEL
+# NOTHING TO DO
+
+account      / account.bank.statement.line / payment_ids (many2many)       : NEW relation: account.payment
+# NOTHING TO DO: Stores payments auto-generated when reconciling lines.
+
+account      / account.bank.statement.line / payment_ref (char)            : NEW required
+# DONE: pre-migration: "payment_ref" is "name" field in v13, also required. Renamed.
+
+account      / account.cash.rounding    / account_id (many2one)         : DEL relation: account.account
+account      / account.cash.rounding    / profit_account_id (many2one)  : NEW relation: account.account, hasdefault
+account      / account.cash.rounding    / loss_account_id (many2one)    : previously in module pos_cash_rounding
+# DONE: pre_migration: "profit_account_id" is a rename of the "account_id" field.
+
+account      / account.chart.template   / account_journal_suspense_account_id (many2one): NEW relation: account.account.template
+# NOTHING TO DO: new features (filled in data on some localization modules)
+
+account      / account.chart.template   / property_cash_basis_base_account_id (many2one): NEW relation: account.account.template
+account      / account.tax.template     / cash_basis_base_account_id (many2one): DEL relation: account.account.template
+# DONE: post-migration: moved from account.tax.template to account.chart.template
+
+account      / account.fiscal.position.account / company_id (many2one)         : NEW relation: res.company, isrelated: related, stored
+account      / account.fiscal.position.tax / company_id (many2one)         : NEW relation: res.company, isrelated: related, stored
+# NOTHING TO DO: filled by Odoo
+
+account      / account.fiscal.year      / company_id (many2one)         : DEL relation: res.company, required, req_default: function
+account      / account.fiscal.year      / date_from (date)              : DEL required
+account      / account.fiscal.year      / date_to (date)                : DEL required
+account      / account.fiscal.year      / name (char)                   : DEL required
+# NOTHING TO DO: preserve data. moved to enterprise.
+
+account      / account.group            / _order                        : _order is now 'code_prefix_start' ('code_prefix')
+account      / account.group            / code_prefix (char)            : DEL
+account      / account.group            / code_prefix_end (char)        : NEW
+account      / account.group            / code_prefix_start (char)      : NEW
+# DONE: pre-migration: renamed code_prefix to code_prefix_start
+# post-migration: assigned same start values to end field
+
+account      / account.group            / company_id (many2one)         : NEW relation: res.company, required, req_default: function, hasdefault
+# NOTHING TO DO
```

Comment:
Maybe change to TODO, because it will be the case that groups are shared between companies, for instance if this Spanish localization was installed in more than one company: https://github.com/odoo/odoo/blob/13.0/addons/l10n_es/data/account_group.xml

Reply:
done, also I added script to solve the issue